src/utils/utils.ts

```diff
@@ -0,0 +1,32 @@
+const polarToCartesian = (
+  centerX: number,
+  centerY: number,
+  arcRadius: number,
+  angleInDegrees: number
+) => {
+  const angleInRadians = (angleInDegrees - 90) * (Math.PI / 180);
+  return {
+    x: centerX + arcRadius * Math.cos(angleInRadians),
+    y: centerY + arcRadius * Math.sin(angleInRadians),
+  };
+};
+
+const describeSlice = (
```

Comment:
I didnt really understand what this function does. explain it to me and then we can maybe think of a different name